.envrc

```diff
@@ -1,7 +1,8 @@
-watch_file devenv.nix
-watch_file devenv.yaml
-watch_file devenv.lock
-watch_file result/bin/devenv
-watch_dir src
-nix-build
-eval "$(./result/bin/devenv print-dev-env)"
+if ! has nix_direnv_watch_file; then
```

Comment:
Due to this if condition my direnv does not download it. You check for nix_direnv and not devenv stuff

Reply:
Yes, this should be different, or at least add a version check similar to my previous PR.
```
if ! has nix_direnv_version || ! nix_direnv_version 2.2.0-devenv; then
  ...
fi

Reply:
I've pushed a version that just always applies the latest direnv integration, don't see a reason not to do this :)